.storybook/main.js

```diff
@@ -1,6 +1,12 @@
 module.exports = {
-  "stories": ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  "addons": ["@storybook/addon-links", "@storybook/addon-essentials", "@storybook/addon-interactions"],
+  "stories": ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/addon-mdx-gfm",
```

Comment:
Reading this add on will be likely be removed in a future version.  Is there a strong reason for including this vs following the recommended instructions? https://www.npmjs.com/package/@storybook/addon-mdx-gfm